portfolio/src/main/java/com/google/sps/data/BookDaoDatastore.java

```diff
@@ -104,10 +105,11 @@ public void update(Book book) {
     private Book parseBook(Entity bookEntity){
         String title = (String) bookEntity.getProperty(TITLE);
         String language = (String) bookEntity.getProperty(LANGUAGE);
-        String description = (String) bookEntity.getProperty(DESCRIPTION);
+        Text descriptionAsText = (Text) bookEntity.getProperty(DESCRIPTION);
```

Comment:
Since we already have books in the datastore, which have description as "String", do you think this will cause a problem during read?